server.js

```diff
@@ -1,23 +1,23 @@
 const express = require('express');
+const app = express(); //moved express() above exphbs;
 const exphbs = require('express-handlebars');
 
-const app = express();
 const PORT = process.env.PORT || 8080;
-const home = require('./router/home');
-const about = require('./router/about');
-const contact = require('./router/contact');
+const home = require('./routes/home'); //misspelled routes with router;
+const about = require('./routes/about'); //misspelled routes with router;
+const contact = require('./routes/contact'); //misspelled routes with router;
 
 app.use(express.static('public'));
-app.engine('.hbs', exphbs({extname: '.hbs', defaultLayout: 'main'}));
+app.engine('.hbs', exphbs({ extname: '.hbs', defaultLayout: 'main' }));
 app.set('view engine', '.hbs');
 
-app.get('/', (req, res) => {
-  res.send('smoke test');
-});
+// app.get('/', (req, res) => {  //don't need this code anymore
```

Comment:
So we need to keep this smoke route around. Can you figure out how to fix the other routes while keeping this one around too?